### PR TITLE
Adds users last read post bookmark information to import

### PR DESCRIPTION
**Pull request by bdharrington7**
implements #134

### CODE REVIEW

File: server/data.js [hunk 1]
```diff
@@ -364,6 +372,10 @@ var async = require('async'),
 		return Data.isImported('_imported:_votes', _vid, callback);
 	};
 
+	Data.isVoteImported = function(_bid, callback) {
```
Comment (akhoury) [inline server/data.js]: shouldnt this be called `isBookmarkImported` ? 
(I know this whole repo is confusing, planning a major refactor right after NodeBB 1.0)

Comment (bdharrington7) [inline server/data.js]: Yep! once again outlining the perils of copy pasta

**Comment by akhoury** [comment]
i feel like Bookmarks are totally unnecessary :P I mean, you're forcing your users to change their passwords, but you still migrated over the last post(s) they read? 

I'm gonna merge it in anyways, but won't publish till next week.

**Comment by bdharrington7** [comment]
Well the password import would be nice, but it's technically infeasible. 

It's really strange to me that it feels like I'm the only one who cares about picking up where I left off in a thread, especially in the case of an infinite-scrolling format. Wouldn't it be a nice user experience to come to a new forum and not have to figure out where you left off in a thread, in an especially active forum that you might not have had time to visit for a while? :) 

Thanks for accommodating my esoteric requests!

**Comment by akhoury** [comment]
> Well the password import would be nice, but it's technically infeasible.

not exactly, we have the _imported_MD5 hash, if you can write the same `comparePassword()` in node, that works exactly like the Vanilla (for example) one - you can then update the user's password with the cleartext in hand - then delete the _imported_MD5 

https://community.nodebb.org/topic/2688/after-migration-passwords/13

**Comment by akhoury** [comment]
that's the ticket for it by the way, ive been very busy to implement new feature though
https://github.com/akhoury/nodebb-plugin-import/issues/64

**Pull request merged**